docs/02-developers/04-quickstart/remix.md

```diff
@@ -0,0 +1,193 @@
+
+## Interacting with the Contract
+
+As shown in the previous step after deploying the contract, you can expand the desired contract and interact with it, calling the available functions by entering the required arguments, if any. Once entered the arguments, click on the method name and, in the event that it’s a writing method, Metamask will prompt you to sign the transaction. If the method is read only, it will show the return value under the method name.
+
+## Verifying the Contract on Rootstock Explorer
+
+To verify the deployed contract on the explorer, go back to Remix file explorer and go to `contracts -> artifacts -> build-info`. In this folder there will be a json file containing the information of the compilation process. You’ll need this in order to verify the contract on the [Rootstock Testnet Explorer](https://explorer.testnet.rootstock.io/). Copy the whole content of the input attribute, i.e what’s inside the curly braces. For e.g, if input value contains `{ value1: 1, value2: 2 }`, file format should be: `{ value1: 1, value2: 2 }`, and paste it in a separate json file in your computer or code editor.
+
+![deployed contract artifact](/img/developers/quickstart/9-remix.png)
+
+Copy the contract's address from the Remix deployment output, you can find the contract address under the **Deployed/Unpinned** Contracts.
+
+![Copy contract address](/img/developers/quickstart/10-remix.png)
+
+Visit the [Rootstock Testnet Explorer](https://explorer.testnet.rootstock.io/) and paste the Contract Address in the search field.
+
+![Paste Contract Address - Rootstock Testnet Explorer](/img/developers/quickstart/11-remix.png)
+
+Click on the Code tab and click the button to **Verify Contract**.
+
+![View Contract Address - Rootstock Testnet Explorer](/img/developers/quickstart/12-remix.png)
+
+![View Verify Explorer Details](/img/developers/quickstart/18-remix.png)
+
+Select **Standard JSON Input** as the verification method and fill the form:
+
+1. Standard JSON input: In the standard JSON input, upload the file you created from the build info of the contract.
+2. Contract name: The name of the contract which you declared it with. Remember that it’s important the file of the contract has the same matching name as the standard-json-input.json. Now, click on add file to upload.
+3. Compiler: This is the compiler version the contract has been compiled with. For example 
+Constructor arguments: The constructor arguments of the contract, if any. If you don’t know the arguments at first, continue with the process and the explorer will attempt to identify the constructor arguments from the bytecode and suggest them, if possible.
+4. ABI encoded arguments: if the arguments provided are ABI encoded, check this option.
+
+Copy only the information inside the input curly braces {}, see [verifying contract on the explorer](#verifying-the-contract-on-rootstock-explorer) and paste the code into a file named `standard-json-input.json`, then click on add file:
```

Comment:
Replace:
> Copy only the information inside the input curly braces {}, see [verifying contract on the explorer](#verifying-the-contract-on-rootstock-explorer) and paste the code into a file named `standard-json-input.json`, then click on add file:

To:
Copy only input value (curly braces included {}), see [verifying contract on the explorer](#verifying-the-contract-on-rootstock-explorer) and paste the value in a new file named `standard-json-input.json`, then click on add file: